src/sentry/api/endpoints/project_app_store_connect_credentials.py

```diff
@@ -199,9 +199,7 @@ def post(self, request: Request, project: Project) -> Response:
             issuer_id=apc_issuer,
         )
         session = requests.Session()
-
         apps = appstore_connect.get_apps(session, credentials)
-
         if apps is None:
```

Comment:
Any specific reason the whitespace was removed?

Reply:
no idea, but i can remove this file 😉